app/controllers/responses_controller.rb

```diff
@@ -22,6 +21,13 @@ def update
 
   private
 
+  def response_by_show_secret_or_id
+    return Response.find_by_show_secret params[:show_secret] if params[:show_secret]
+    return Response.find params[:id] if params[:id]
+
```

Comment:
Hier moet nog worden gekeken of er wel een ResponseFinished event bestaat met dezelfde response_uuid als het InvitationAccepted event. Anders kun je scores opvragen die niet door de invuller zijn bevestigd.